courseaffils/columbia.py

```diff
@@ -145,7 +145,7 @@ def to_string(cdict):
     @staticmethod
     def to_dict(wind_string):
         wind_match = re.match(
-            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\d{3})'
+            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\w{3})'
```

Comment:
Hmm... I think I actually need to change this in all three occurrences of the regex then. Somehow, when I changed this one, my test started passing, but it does seem like the new patter is now valid, so I think it needs to be updated everywhere.